### PR TITLE
Update annotation stats info

### PR DESCRIPTION
**Pull request by smoreface**
Update with more details about available metrics and a link to selecting a matching function

### CODE REVIEW

File: docs/source/guide/stats.md [hunk 1]
```diff
@@ -22,101 +24,102 @@ Task agreement shows the consensus between multiple annotators when labeling the
...
 
-Matching scores are used to determine whether two given annotations for a task, represented by `x` and `y` in this example, match. 
-- If both `x` and `y` are empty annotations for a task, the matching score is `1`.
+The matching score assesses the similarity of annotations for a specific task. For example, for two given annotations `x` and `y`, a matching function that performs a naive comparison of the results works like the following:
+- If both `x` and `y` are empty annotations, the matching score is `1`.
 - If `x` and `y` share no similar points, the matching score is `0`. 
-- If there are different labeling types used in the annotations in `x` and/or `y`, partial matching scores for each data labeling type are averaged.
-- For categorical task labeling, such as those using the Choices tag, Cohen's Kappa index is computed if specified in the project settings. 
+- If different labeling types are used in `x` and `y`, the partial matching scores for each data labeling type are averaged.
+- For categorical or classification labeling tasks, such as those using the Choices tag, Cohen's Kappa index is computed if specified in the project settings.
```
Comment (KonstantinKorotaev) [inline docs/source/guide/stats.md:72]: Cohen's Kappa is disabled

File: docs/source/guide/stats.md [hunk 2]
```diff
@@ -22,101 +24,102 @@ Task agreement shows the consensus between multiple annotators when labeling the
...
-For more about viewing agreement in Label Studio Enterprise, see [Verify model and annotator performance](quality.html#Verify-model-and-annotator-performance)
+For more about viewing agreement in Label Studio Enterprise, see [Verify model and annotator performance](quality.html#Verify-model-and-annotator-performance).
+
+## Agreement method
+
+The agreement method defines how [matching scores](stats.html#Matching-score) across all annotations for a task are combined to form a single inter-annotator agreement score.
+
+There are several possible methods you can specify on the **Annotation Settings** for a project.
+
+### Complete linkage
```
Comment (KonstantinKorotaev) [inline docs/source/guide/stats.md:35]: @niklub lost in LSE 2.0
Where should it be used?

Comment (niklub) [reply docs/source/guide/stats.md:35]: These are matching score aggregation methods. We left only with "No grouping" due to complexity reasons. Not needed anymore in doc

Comment (smoreface) [reply docs/source/guide/stats.md]: Perfect, I updated accordingly

**Comment by smoreface** [comment]
@niklub / @KonstantinKorotaev  please re-review thanks!

**Review by KonstantinKorotaev — APPROVED** [review]

**Pull request merged**